modules/docs/arrow-docs/docs/_data/menu.yml

```diff
@@ -257,6 +257,9 @@ options:
       - title: Each
         url: /docs/optics/each/
 
+  - title: Generic
+    url: /docs/generic/
+
```

Comment:
Can you please add one nested option called `Intro` or something similar? We can fill it in tomorrow.

Reply:
I'm not sure clicking this menu option will go to `/docs/generic/` as described below. I believe it needs a submenu.